recipes-support/lvrt-logging/files/logging_paths.ini

```diff
@@ -0,0 +1,7 @@
+[Paths]
```

Comment:
You have these changes in https://github.com/ni/meta-nilrt/pull/839. Why duplicate?

Reply:
This was a branching mistake, I have fixed this now.